src/index.ts

```diff
@@ -8,7 +8,7 @@ import { Observer } from './Observer';
 
 let Accessory: any;
 
-export default function(homebridge: any) {
+export default function (homebridge: any) {
```

Comment:
This space should be removed

Reply:
Yeah, I thought so too but somehow vscode kept putting it back.. I'm guessing it's some eslint rule triggering....

Reply:
Actually this is correct now that I think about it, it's an anonymous function which should have a space there (normally the function name would be there but since it's anonymous there's just a space.) However, how about making this an arrow function instead?